suite-native/module-authorize-device/src/screens/passphrase/PassphraseConfirmOnTrezorScreen.tsx

```diff
@@ -53,6 +66,44 @@ export const PassphraseConfirmOnTrezorScreen = () => {
         }
     }, [device, dispatch, isDeviceConnectedAndAuthorized, isDiscoveryActive, navigation]);
 
+    useEffect(() => {
+        // User has canceled the authorization process on device (authorizeDeviceThunk rejects with auth-failed error)
+        if (hasAuthorizationFailed || hasVerifyCancelledError) {
```

Comment:
Should canceling on device when confirming passphrase second time cancel whole flow?

Reply:
I believe so

Reply:
Just to clarify - it's the same as the close button on the screen in app, Trezor device just doesn't display warning sheet, so we kill the flow.